raiden/network/proxies/token_network.py

```diff
@@ -2529,9 +2526,9 @@ def _settle(
 
     def events_filter(
         self,
-        topics: List[str] = None,
-        from_block: BlockSpecification = None,
-        to_block: BlockSpecification = None,
+        topics: Optional[List[Optional[str]]],
+        from_block: BlockSpecification,
```

Comment:
You probably did check, but I am just making sure. Have you checked if `from_block` and `to_block` are omitted from any call to this function?

Reply:
Good point, was able to inline this completely